package.json

```diff
@@ -4,7 +4,14 @@
   "description": "Contains the functionality of the consumer and provider of a Webrtc stream in the cloud",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "bootstrap": "lerna bootstrap --hoist",
+    "build": "lerna run build",
+    "test": "lerna run test",
+    "lint": "lerna run lint",
+    "start": "lerna exec --scope @browserd/stream-provider -- npm run start",
+    "clean": "lerna clean --yes",
+    "dist": "lerna exec --scope @browserd/stream-provider -- npm run dist",
+    "magic": "npm run bootstrap && npm run build && npm run start"
```

Comment:
surface all the npm things one is likely to do to the top, since there's no real value in operating on a single-package level rn (you'd need to build shared to build stream-provider, etc). 

also add magic to just do the "right thing" for now. When we add a runnable stream-consumer this will need to get renamed to `magic-provider` or similar.

Reply:
it is cleaner to have all the lerna stuff wrapped within the scripts. definitely makes things easier